kit/src/components/user/mod.rs

```diff
@@ -34,7 +34,7 @@ pub fn get_time_ago(cx: &Scope<Props>) -> String {
     let duration: std::time::Duration = match c.to_std() {
         // for the sidebar, don't want the timestamp to increment a few seconds every time the typing indicator comes over.
         // prevent this by rounding down and giving a duration in minutes only.
-        Ok(d) => std::time::Duration::from_secs(d.as_secs() / 60),
+        Ok(d) => std::time::Duration::from_secs(d.as_secs() / 60 * 60),
```

Comment:
could you just remove the divide by 60 here?

Reply:
i thought the point was to not have it as seconds? this way it only displays at most as minutes.
since e.g. 40 sec -> 0 sec -> "now"
or 65 -> 60 -> 1 min

Reply:
oh, cool. I did not know that

Reply:
thank you :)